app/assets/stylesheets/custom.css.scss

```
@@ -0,0 +1,42 @@
+@import 'bootstrap/dist/css/bootstrap';
```

You can import the SCSS files of Bootstrap directly.

```suggestion
@import 'bootstrap/scss/bootstrap';
```

see https://getbootstrap.com/docs/4.0/getting-started/theming/#importing

---

Rename file to `custom.scss`? Or is there any reason for the `.css.scss`?

---

Hey @aboutsimon cool, good question... To be honest, I have no idea. This is the code we are taught in the course. What is the difference between `scss` und `css.scss`?

---

Well, `.scss` is the extension of the [css preprocessor SASS](https://sass-lang.com/) (don't be confused, by sass and scss, they are just two different flavors of SASS, with SCSS being the more popular one). `.css` just stand's for plain old css. The `*.css.scss` is probably just a weird Rails default?!

---

OK, so should I make the changes you suggest or leave it for now? I don't mind which way :)

---

I would leave the filename as is and try changing the import statement to `@import '~bootstrap';` as suggested above. That way, you could [theme your bootstrap instance](https://getbootstrap.com/docs/4.0/getting-started/theming/), if you want.

---

but don't get hung up on that, this is not a merge blocker ;)